### ISSUE TITLE
Seedlet: keyboard navigation issues

### ISSUE THREAD

**Issue by jffng** [Automattic/themes#2284]
There are a handful of keyboard navigation issues that need addressed before the theme can be accepted in the dotorg themes directory.

- [x] The wp admin bar styles should not be affected by your styles (focus for example)
- [x] Mobile: Visual order is wrong, burger button focus should be before the logo focus (mobile)
- [x] Mobile: Within the menu, focus should be trapped inside the modal. Once you tab past the last menu item, focus should go on the close button or close the menu. Once you shift+tab on the first element focus should go on the close button (and if you shift tab on this, it should focus the last element in the menu).

**Comment by kjellr** [comment]
This is all set, as per #2305
